lib/swr/use-viewer.ts

```
@@ -1,35 +1,96 @@
   const { data: viewer, error } = useSWR<ViewerWithViews>(
-    teamId && id && `/api/teams/${teamId}/viewers/${id}`,
+    teamId && id ? `/api/teams/${teamId}/viewers/${id}?${paginationParam}${sortingParam}` : null,
     fetcher,
+    {
+      revalidateOnFocus: false,
+      dedupingInterval: 30000,
+      revalidateIfStale: false,
+    }
   );
 
+  const shouldFetchDurations = viewer && viewer.views && viewer.views.length > 0;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)